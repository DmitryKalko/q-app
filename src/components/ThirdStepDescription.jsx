import React from "react";
import thirdStep from "./svg/third-step.svg";

const ThirdStepDescription = ({ lightTheme }) => {
  return (
    <div className="step">
      <p style={lightTheme ? { color: "white" } : { color: "#6d6d6d" }}>
        How do you want to receive notifications?
      </p>
      <img src={thirdStep} alt="firstStep" />
    </div>
  );
};
export default ThirdStepDescription;
