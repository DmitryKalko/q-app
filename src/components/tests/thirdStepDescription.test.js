import { shallow } from "enzyme";
import React from "react";
import ThirdStepDescription from "../ThirdStepDescription";

describe('about work FirstStepDescription component', () => {
    it('should render FirstStepDescription component', () => {
        const component = shallow(<ThirdStepDescription/>)
        expect(component).toMatchSnapshot();
    });
});