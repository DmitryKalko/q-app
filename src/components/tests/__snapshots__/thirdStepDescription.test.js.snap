// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about work FirstStepDescription component should render FirstStepDescription component 1`] = `
<div
  className="step"
>
  <p
    style={
      Object {
        "color": "#6d6d6d",
      }
    }
  >
    How do you want to receive notifications?
  </p>
  <img
    alt="firstStep"
    src={
      Object {
        "process": [Function],
      }
    }
  />
</div>
`;
